test/test_pdo.py

```diff
@@ -55,10 +65,23 @@ def test_bit_mapping(self):
         self.assertEqual(node.tpdo[0x2002].raw, 0xf)
         self.assertEqual(node.pdo[0x1600][0x2002].raw, 0xf)
 
-    def test_save_pdo(self):
-        node = canopen.Node(1, EDS_PATH)
-        node.tpdo.save()
-        node.rpdo.save()
+    def test_pdo_save(self):
+        self.node.tpdo.save()
+        self.node.rpdo.save()
+
+    @unittest.skipIf(canmatrix is None, "Requires canmatrix dependency")
```

Comment:
Can we do a simple runtime check and catch the exception instead of this decorator using a sentinel value (`None`)?  Something like (from the top of my head, no clue about the unittest skipping function):

    try:
        _ = canmatrix.canmatrix
    except NameError:
        unittest.skip()

If a simple boolean condition is the only way to cleanly skip the test, let's do that in the import exception block above, but rather not with the `canmatrix` name.

Reply:
Sure!